package.json

```diff
@@ -44,7 +44,8 @@
     "react-helmet": "^5.2.0",
     "react-inspector": "^2.3.0",
     "react-intl": "^2.4.0",
-    "react-router-dom": "^4.3.1",
+    "react-overlays": "^0.8.3",
+    "react-router-dom": "^4.2.2",
```

Comment:
Hm.. why `4.2.2` and not `4.3.1`?

Reply:
That wasn't supposed to happen. OOps. THanks!!!!